Neos.Neos/Classes/Controller/Module/Administration/UsersController.php

```diff
@@ -84,13 +84,21 @@ protected function initializeAction()
     /**
      * Shows a list of all users
      *
+     * @param string $searchTerm
      * @return void
      */
-    public function indexAction()
+    public function indexAction(string $searchTerm = '')
     {
+        if (empty($searchTerm)) {
```

Comment:
In the second column of the list, the username is shown, but not searchable. Would it be possible to also search by the accounts username?

Reply:
Done